scripts/beachball/getVNextChangelogGroups.ts

```diff
@@ -0,0 +1,37 @@
+import { BeachballConfig } from 'beachball';
+import { AllPackageInfo, getAllPackageInfo } from '../monorepo/index';
+
+/**
+ * Generates grouped changelog for vNext
+ * @returns Groupped changelog configuration
+ */
+export function getVNextChangelogGroups(): BeachballConfig['changelog']['groups'][0] {
```

Comment:
I think you should be able to use `VersionGroupOptions` directly? (If it's not exported we should fix that.)

Reply:
It's `ChangelogGroupOptions` but you're right, the appropriate type is there 👍